generators/gen_reactnative.py

```diff
@@ -103,6 +106,7 @@ def __init__(self, output_folder: str):
 
     def generate(self, definitions: list):
         self._generate_ts_index(definitions)
+        self._generate_ts_dd_foundation(definitions)
```

Comment:
What's the purpose of splitting things between index and dd_foundation? Also not a fan of the `dd_foundation` name which is not very generic (could be `foundation` even though it's not a very explicit name).

Reply:
the split is needed because otherwise there is a `require` circular chain, which was fixed here https://github.com/DataDog/dd-sdk-reactnative/pull/35.